_content/tour/variables.article

```diff
@@ -41,7 +41,7 @@ and pointers for a given architecture. For example:
 - 32 bit arch: word size is 4 bytes of memory allocation
 - 64 bit arch: word size is 8 bytes of memory allocation
 
-This is important because Go has internal data structures (maps, channels, slices,
+This is important because Go has internal data structures (strings, maps, channels, slices,
```

Comment:
Adding string here would be wrong. A string is considered a built-in or scalar type. When a string is set to its zero value it is empty. When the internal data types are set to their zero value they are nil. There is a semantic difference here.